### ISSUE TITLE
Upgrading to 0.6.0 do not process .scss files anymore

### ISSUE THREAD

**Issue by kuon** [brunch/postcss-brunch#18]
After upgrading to `postcss-brunch` `0.6.0` my `.scss` files are no longer processed by `postcss`.

If I remame my `.scss` file to `.css` postcss properly processes my files.

**Comment by razor-x** [comment]
Just curious if anyone is looking into this or has an idea what might be the cause.

**Comment by 32bitkid** [comment]
I _think_ it's because during the refactor to 0.6.0 the plugin structure changed from being a css optimizer, using the `optimize()` hook, to a css _compiler_, using the `compile()` hook. So, now it only will compile sources that _start_ as a css file.

**Comment by kuon** [comment]
So, what should we do to have it process `.scss`?

**Comment by 32bitkid** [comment]
i need to look at the new brunch plugin structure/tree more to understand what the right change is to make.

_Before_ the output of `sass-brunch` would get pumped into the input of `postcss-brunch` and then written to the output directory. _Now_, `sass-brunch` and `postcss-brunch` are more like siblings, just that `sass-brunch` reads sass|scss files while `postcss-brunch` reads `css` files. They don't chain anymore.

So, I need to look to see if there is a better way to wire those together/understand why it changed from a `optimize()` to a `compile()` for places where you want to compile _into_ css (less,scss, myth) and then have it transformed by `postcss`.

**Comment by lydell** [comment]
This bit me today, too.

I was under the impression that all my .scss files were getting vendor prefixes with postcss+autoprefixer. Turns out they were not! (Things just happened to work for me anyway, because I mostly test in bleeding-edge browsers. Then, when I finally didn’t – boom! Lots of missing styling :) )

The way I did this in my old [autoprefixer-brunch](https://github.com/lydell/autoprefixer-brunch/blob/master/src/index.coffee) plugin is:
- Don’t use `compile()`, but `optimize()`.
- add `defaultEnv: '*'` so that the plugin is run not only in the 'production' env.
